src/Metadata/Resource/Factory/OperationResourceMetadataFactory.php

```diff
@@ -81,6 +82,13 @@ public function create(string $resourceClass): ResourceMetadata
             $resourceMetadata = $this->normalize(false, $resourceClass, $resourceMetadata, $itemOperations);
         }
 
+        if ($this->needsDefaultGetOperation($resourceMetadata)) {
+            $resourceMetadata = $resourceMetadata->withItemOperations(array_merge(
+                $resourceMetadata->getItemOperations(),
+                ['get' => ['method' => 'GET', 'read' => false, 'output' => ['class' => false], 'controller' => NotFoundAction::class]])
```

Comment:
I believe you've made a mistake here, it's supposed to be `null`:

https://github.com/api-platform/core/blob/81d276c33c871ee45605222d474181a503cdf1bd/src/Metadata/Resource/Factory/InputOutputResourceMetadataFactory.php#L95-L97